src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,9 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
 public interface MachineProducer {
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/pecs/pecs.html#parameterize-machineproducer-in-such-a-way-that-any-implementation-not-using-machine-or-its-subtype-could-not-be-created